packages/web/pages/player/[username].tsx

```diff
@@ -500,11 +535,9 @@ export const getStaticProps = async (
   }
 
   const player = await getPlayer(username);
-  const personalityInfo = await getPersonalityInfo();
```

Comment:
The reason I had moved this here was to reduce load time on frontend, if this static data could be loaded on server side itself. But never mind :)

Reply:
I figured that out when I saw the same pattern for skills.

The problem was it was drilled through three or four components completely unrelated to the color disposition, and was just out of place. Maybe we could get the same effect using an `Atom` or `Context` in the future without the prop drilling.